libcxx/test/libcxx/utilities/utility/pairs/pairs.pair/trivially_copyable.compile.pass.cpp

```diff
@@ -0,0 +1,32 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03 || !clang && (c++11 || c++14 || c++17)
```

Comment:
```suggestion
// UNSUPPORTED: gcc && (c++11 || c++14 || c++17)
// XFAIL: !libcpp-abi-whatever-trivially-copyable-pair
```

Those should be the only configurations where this test doesn't pass. Or even better, pin down the ABI of pair all the time in this test.